### PR TITLE
Add more glossary terms

### PR DESCRIPTION
**Pull request by zi-yii**
#37

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -517,9 +517,12 @@ Use case ends.
 ### Glossary
 
 * **Mainstream OS**: Windows, Linux, Unix, MacOS
-* **Claims**: A formal request by a client for reimbursement for losses that are covered by specific insurance plans.
+* **Client**: A person who purchased at least one insurance plan from the insurance agent.
```
Comment (matthewyip1511) [inline docs/DeveloperGuide.md:520]: What about potential customers? We might still consider them as "clients" even though they have not yet purchased anything

**Review by matthewyip1511 — CHANGES_REQUESTED** [review]

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (matthewyip1511) [reply docs/DeveloperGuide.md:520]: What about

Client: A customer that the user is speaking to. Has either purchased a plan, or is keen to do so

Comment (zi-yii) [reply docs/DeveloperGuide.md:520]: Nice, thanks! Will make the edits

**Review by RezwanAhmed123 — APPROVED** [review]
LGTM, logical terms were added to the glossary relevant to project.

**Comment by RezwanAhmed123** [comment]
fixes #37

**Pull request merged**